libexec/phpenv-init

```diff
@@ -56,6 +56,9 @@ if [ -z "$print" ]; then
   ksh )
     profile='~/.profile'
     ;;
+  fish )
+    profile='~/.profile'
```

Comment:
default startup config path of fish-shell is `~/.config/fish/config.fish`

## See also

* fish document: http://fishshell.com/docs/current/tutorial.html#tut_startup
* rbenv code: https://github.com/rbenv/rbenv/blob/master/libexec/rbenv-init#L61